src/detect-flow.c

```diff
@@ -227,6 +227,9 @@ static DetectFlowData *DetectFlowParse (DetectEngineCtx *de_ctx, const char *flo
                 if (fd->flags & DETECT_FLOW_FLAG_ESTABLISHED) {
                     SCLogError("DETECT_FLOW_FLAG_ESTABLISHED flag is already set");
                     goto error;
+                } else if (fd->flags & DETECT_FLOW_FLAG_NOT_ESTABLISHED) {
+                    SCLogError("DETECT_FLOW_FLAG_NOT_ESTABLISHED flag is already set");
```

Comment:
error message should be different, right ?

Reply:
Thank you, I have changed the error message. Is it OK now?